tsconfig.json

```diff
@@ -2,7 +2,6 @@
   "compilerOptions": {
     "module": "commonjs",
     "target": "ES6",
-    "noImplicitAny": true,
```

Comment:
Hurts me more than it hurts anyone but I would've had to make a crapton of changes just to get it to build and given the time constraints we're under...made a judgement call.